### PR TITLE
Rename to google-native

### PR DESCRIPTION
**Pull request by mikhailshilkov**

### CODE REVIEW

File: provider/pkg/gen/schema.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ func PulumiSchema() (*schema.PackageSpec, *resources.CloudAPIMetadata, error) {
 	}
 
 	csharpNamespaces := map[string]string{
-		"gcp-native": "GcpNative",
+		"google-native": "GcpNative",
```
Comment (lblackstone) [inline provider/pkg/gen/schema.go:53]: Should the namespace change as well?

**Review by viveklak — APPROVED** [review]

File: provider/pkg/gen/schema.go (hunk 1 above)
Comment (lblackstone) [reply provider/pkg/gen/schema.go:53]: @viveklak @mikhailshilkov ?

Comment (viveklak) [reply provider/pkg/gen/schema.go:53]: Sorry I hammered this through to fix a build for docs locally.  I will fix this shortly. Thanks for pointer @lblackstone

Comment (viveklak) [reply provider/pkg/gen/schema.go:53]: Follow-up here: https://github.com/pulumi/pulumi-google-native/pull/50

**Pull request merged**